scripts/fast-reboot

```diff
@@ -659,6 +631,15 @@ if [[ "$REBOOT_TYPE" = "fast-reboot" ]]; then
         unload_kernel
         exit "${EXIT_COUNTERPOLL_DELAY_FAILURE}"
     fi
+
+    # Clear all routes except of default routes for faster reconciliation time.
```

Comment:
Since this is a delete call (cannot recover from here), shouldn't this step be executed after `set +e` ?

Reply:
Good catch! modified.

will update also in https://github.com/sonic-net/sonic-utilities/pull/2365 (PR for 202205)